src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -96,8 +100,11 @@ window.qBittorrent.DynamicTable ??= (() => {
             const tableFixedHeaderDiv = $(this.dynamicTableFixedHeaderDivId);
 
             const tableElement = tableFixedHeaderDiv.querySelector("table");
+            const rerender = () => { this.rerender(); };
```

Comment:
Would this work?
```suggestion
            const rerender = this.rerender;
```

Reply:
It's not magic, that's how arrow functions work in JS.
The function below uses a regular function for some reason, so `this` context is bind to the emitter, not for the surrounding scope.
This whole thing could be avoided by replacing it by an arrow function.

Reply:
It doesn't matter. See https://github.com/qbittorrent/qBittorrent/pull/22502#discussion_r2026855048.